### DISCUSSION TITLE
말말말

### DISCUSSION THREAD

**Discussion by sunnylina** [Jung110/kaggleteam#17] [General]
트레인이랑 안트레인이랑 머징 해서 안팔린 아이템도 추가해서 머머머하는거해보기
안트레인이 뭐엿지
주기성땡
아리마 시계열
축 시간별로 요러ㅏㅎ게 나누다 보면 달별로 2014년 이후 2013년이 나오는게 아니고 순서가 정해져있고 데이터셋을 정리 해야 하고 
데이블럭을 정리 해서 뭘해야 하고 다른 5개 있는 다른 변수들은 안씉다고?
시계열의 특징
템플릿 큰틀 동일 하게 배포 전처리는 각자 하고  서로서로 각자 한거 공유 하기,,,,,,,,,,,,,,,,,,
시계열은 최근 3개월치가 더 영향력이 있다 

오옹 어떻게 나눌것인가
3년전의 유행이랑 올해 데이터랑 달라지는 추이를 예상하고 그거에 따라 기준을 어떻게 만든

**Comment by sunnylina** [comment]
연도 별로 전처리를 다시 해야한다고?!
시게열 데이터
검증방법
어떻게 나눌건지
어떤식으로 나눠야 한다.
덩어리가 크면 클수록 방향성이 둔해진다
데이터 6개월 상반기 하반기 하면 6번은 나오니까
train test valuation

**Comment by sunnylina** [comment]
그 교차 검증이 그 교차 검증이었습니다.
shop id item id 
쌩으로
만든다음에 추가로 
추가
추가추가
맨마지막 최종값은 씨엔티겠죠
달별로 잘라야 하고 날짜별로는 절대 못자르고 하다 못해 분기 별로

**Comment by sunnylina** [comment]
일단 6개월로 ?
이거로 하면 2개월 단위로 잘라보고 이거가 뭐지,,,,,
결국 교차 검증을 하기위해서 valuation set을 지정 하는것

**Comment by sunnylina** [comment]
arma/arima

**Comment by Jung110** [comment]
최신 개월로 valid 하는 것이 제일 정확 하다 면 test 전 달을 valid로 하고 나머지는 train으로 하면 되지 않을 까요?
